### ISSUE TITLE
 fatal exception after update to 17.0.2

### ISSUE THREAD

**Issue by lwt-pressy** [EDDiscovery/EDDiscovery#3475]
crashing after update with:
An unhandled fatal exception has occurred.

```
==== BEGIN ====
System.ArgumentException: Der Delegat einer Instanzenmethode kann kein "this" von NULL haben.
   bei System.MulticastDelegate.ThrowNullThisInDelegateToInstance()
   bei EliteDangerousCore.JournalEntry.CreateJournalEntries(List`1 tabledata, Func`1 cancelRequested, Action`1 progress) in C:\Code\EDDiscovery\EliteDangerousCore\EliteDangerous\Journal\JournalEntry.cs:Zeile 0.
   bei EliteDangerousCore.EDSM.EDSMLogFetcher.Process(List`1 edsmlogs, DateTime logstarttime, DateTime logendtime) in C:\Code\EDDiscovery\EliteDangerousCore\EliteDangerous\EDSM\EDSMLogFetcher.cs:Zeile 0.
   bei EliteDangerousCore.EDSM.EDSMLogFetcher.FetcherThreadProc() in C:\Code\EDDiscovery\EliteDangerousCore\EliteDangerous\EDSM\EDSMLogFetcher.cs:Zeile 131.
   bei System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   bei System.Threading.ThreadHelper.ThreadStart()
===== END =====
```

**Comment by lwt-pressy** [comment]
Some Observation 17.0.2 starts for some reason getting the data in 2016. 

> [2024-01-27T11:24:01.858Z] EDSM Log Fetcher ask for week up to 03.08.2015 12:00:00
>[2024-01-27T11:24:01.858Z] HTTP GET to https://www.edsm.net/api-logs-v1/get-logs?showId=1&showCoordinates=1&apiKey=xxx&commanderName=Pressy&endDateTime=2015-08-03+12%3a00%3a00 Thread 'EDSM Log Fetcher'
>Content-Type:application/json; charset=utf-8
>Accept-Encoding:gzip,deflate

Also Database seems ok, because going back to 17.0.1 just works fine.

**Comment by Eahlstan** [comment]
Thanks for the report, I was able to reproduce it. Falling back to 17.0.1 was the correct thing to do in this case as we have to update the release to fix this.

**Comment by robbyxp1** [comment]
Fixed in 17.0.3

**Comment by lwt-pressy** [comment]
thx, update to 17.0.3 works like a charm :)
